docs/resources/Channel.md

```diff
@@ -266,6 +268,7 @@ Represents a message sent in a channel within Discord.
 | components?               | Array of [message components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object)                                              | sent if the message contains components like buttons, action rows, or other interactive components                                               |
 | sticker_items?            | array of [message sticker item objects](#DOCS_RESOURCES_STICKER/sticker-item-object)                                               | sent if the message contains stickers                                                                                                            |
 | stickers?                 | array of [sticker](#DOCS_RESOURCES_STICKER/sticker-object) objects                                                                 | **Deprecated** the stickers sent with the message                                                                                                |
+| position?                 | integer                                                                                                                            | position of the message in a forum post when it was sent, it will not change upon any message deletion in the parent post                    |
```

Comment:
I think this is for all threads, not just forum posts, right?
I'd suggest: "A generally increasing integer (there may be gaps or duplicates) that represents the approximate position of the message in a thread."